connector/connect/common/src/main/protobuf/spark/connect/commands.proto

```diff
@@ -209,6 +209,15 @@ message WriteStreamOperationStart {
     string path = 11;
     string table_name = 12;
   }
+
+  optional Foreach foreach = 13;
+}
+
+message Foreach {
```

Comment:
`Foreach` seems too short and does not convey much context. How about `StreamingForeachSink` or `StreamingForeachWriter`?